probes/hasWebhooks/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: hasWebhooks
+short: Checks whether the project has any workflows
+motivation: >
+  This is more of a utility probe to help users determine whether they should attempt to score the security of a given projects webhook settings.
+implementation: >
+  The probe checks the number of webhooks from the raw results.
+outcome:
+  - If a project has webhooks, the probe returns a number of OutcomePositives equal to the number of webhooks that the project has.
+  - If a project does not have any webhooks, the probe returns a single OutcomeNegative.
+remediation:
+  effort: Low
+  text:
+    - Add webhooks to the project.
```

Comment:
that does not sound right :) We dont want to force users to add webhooks if they don't have any. Do we even need this probe? What if we could return NotApplicable in the other probes's results?

Reply:
> What if we could return NotApplicable in the other probes's results?

This is https://github.com/ossf/scorecard/issues/2928. 

In this case, the pros are that we could remove the `hasWebhooks` probe and use `Not Applicable` to specify that a project does not have webhooks. This would be helpful for this probe.

Reply:
makes sense to me, as discussed in the weekly sync.

Reply:
Removed `hasWebhooks` in https://github.com/ossf/scorecard/pull/3522#issuecomment-1811514146